draft-ietf-tls-tls13.md

```diff
@@ -2455,38 +2393,59 @@ alert.
 #### Early Data Indication
 
 In cases where TLS clients have previously interacted with the
-server and the server has supplied a known configuration, the client
+server and the server has supplied a server configuration, the client
```

Comment:
"server configuration" -> "ServerConfiguration {{server-configuration}}"

Using the message name and adding a cross-reference link makes learning how this stuff works easier. Also easier to search the document for something specific if the naming is consistent.